.eslintrc.json

```diff
@@ -1,22 +1,28 @@
 {
-  "env": {
+    "env": {
       "browser": true,
-      "es2021": true
-  },
-  "extends": [
-      "eslint:recommended",
-      "plugin:react/recommended"
-  ],
-  "parserOptions": {
+      "es6": true,
+      "jest": true
+    },
+    "parser": "@babel/eslint-parser",
+    "parserOptions": {
       "ecmaFeatures": {
-          "jsx": true
+        "jsx": true
       },
-      "ecmaVersion": 12,
+      "ecmaVersion": 2018,
       "sourceType": "module"
-  },
-  "plugins": [
-      "react"
-  ],
-  "rules": {
+    },
+    "extends": ["airbnb", "plugin:react/recommended", "plugin:react-hooks/recommended"],
+    "plugins": ["react"],
+    "rules": {
+      "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+      "react/react-in-jsx-scope": "off",
+      "import/no-unresolved": "off",
+      "no-shadow": "off"
+    },
```

Comment:
- Good job with this, @SaboorSohaib :clap: :clap: I was just about adding the needed dependencies.
- [ ] [OPTIONAL] It's imperative to communicate effectively when working on a group project, as this will help you to save a lot of time resolving got conflicts.  I was working on this and It could have resulted in another potential conflict, depriving us of adequate time to focus on the project features. :wink:

Reply:
### Hi @AbayomiOlaoye :wave: :wave: :wave:

Thank you very much for your golden advice. However, I contacted @MudasirSherwani and he told me to fix linter error.

Thank you very much again.